### DISCUSSION TITLE
How can i use my own map?

### DISCUSSION THREAD

**Discussion by Berkanktk** [themustafaomar/jsvectormap#116] [Q&A]
Is there any way that i can specify a path to my own [world_merc_en.js](https://gist.github.com/Berkanktk/15359fc23833728039036d986318430a) file? @themustafaomar

**Comment by themustafaomar** [comment]
Don't know how I didn't see that, sorry for the late response.

You just need to import it after jsvectormap, you're done, just tell it which map you want to use as below.

```js
const map = new jsVectorMap({
  map: 'world_merc_en'
})
```

**Reply by Berkanktk** [reply]
No worries, I figured it out real fast. But thanks though! Should've had closed the issue by now:)
